src/TTrigger.h

```diff
@@ -153,7 +153,7 @@ class TTrigger : public Tree<TTrigger>
     bool            mModuleMasterFolder;
 private:
 
-                                           TTrigger(){};
+                                           TTrigger(){}
```

Comment:
Fixes a minor compiler warning - Clang analyser notices and yellow triangles the extraneous `;` :wink: